support.md

```diff
@@ -0,0 +1,8 @@
+# Support
```

Comment:
This no longer represents our current support process... Perhaps cian can help writing this up.